packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -243,17 +244,19 @@ void main() {
     });
     await tester.pump();
 
-    expect(log, <MethodCall>[
-      const MethodCall('TextInput.setEditingState', const <String, dynamic>{
-        'text': 'Wobble',
-        'selectionBase': -1,
-        'selectionExtent': -1,
-        'selectionAffinity': 'TextAffinity.downstream',
-        'selectionIsDirectional': false,
-        'composingBase': -1,
-        'composingExtent': -1,
-      }),
-    ]);
+    expect(log, hasLength(1));
+    expect(log.single, isMethodCall(
+          'TextInput.setEditingState',
+          arguments: const <String, dynamic>{
+            'text': 'Wobble',
+            'selectionBase': -1,
+            'selectionExtent': -1,
+            'selectionAffinity': 'TextAffinity.downstream',
+            'selectionIsDirectional': false,
+            'composingBase': -1,
+            'composingExtent': -1,
+          },
+    ));
```

Comment:
Prettier indentation with trailing comma here?

Reply:
This was intentional to make the parens symmetric with the opening line.